net/nut/files/nut-sendmail-notify

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+(cat - | /usr/sbin/sendmail root ) <<EOF
+From: root
+To: root
+Subject: UPS $NOTIFYTYPE Notification
+
+$1
+.
+EOF
```

Comment:
Guess there is no need to use subshell.  I see no reason why sendmail cannot work directly with here document ;)